src/services/WebSocketPolyfill.js

```diff
@@ -60,7 +56,16 @@ export default function initWebSocketPolyfill(syncService, fileId, initialSessio
 					}
 				},
 			})
-			syncService.open({ fileId, initialSession })
+
+			syncService.open({ fileId, initialSession }).then((data) => {
+				if (syncService.hasActiveConnection) {
```

Comment:
I decided to refactor this slightly as we then can just always rely on the open method to return the relevant data, in both first connects and reconnects